Gemfile

```diff
@@ -49,6 +49,10 @@ gem 'sentry-ruby'
 gem 'sentry-rails'
 gem 'sentry-sidekiq'
 
+gem 'faraday'
+gem 'faraday_middleware'
```

Comment:
@Jullianos @AndriiMysko `faraday_middleware` is deprecated.
Why you are not using `rest-client` instead of `faraday`?